README.md

```diff
@@ -21,30 +21,30 @@ Duplicate the `.env` file and rename it as `.env.local` to add your local secret
 Install the project dependencies
 
 ```bash
-npm install
```

Comment:
i see a lot of projects show the instructions for multiple package managers. Is pnpm required or will both work? We'd only publish one lock file I assume so maybe that is an issue...

Reply:
I think that PNPM is required. I'll update this doc.

Reply:
I'm curious to see how other projects deal with this. Any examples? I've seen other projects show examples of how to install their package using various package managers. But I'm not sure that I've seen them show how to set up the dev environment using multiple managers.